frontend/src/app/projects/models/backup-settings/pipeline-runs/service/pipeline-run.service.ts

```diff
@@ -92,17 +113,58 @@ export class PipelineRunService {
   loadPipelineRuns(
     projectSlug: string,
     modelSlug: string,
-    pipelineID: number
-  ): Observable<PipelineRun[]> {
+    pipelineID: number,
+    page: number,
+    size: number
+  ): Observable<Page<PipelineRun> | undefined | 'loading'> {
```

Comment:
Both `load...` functions here should have `void` as return type and instead of using `tap` you can then directly put this into the `subscribe` block. Please check the usages of both functions and removed the `subscribe()` from them